package.json

```diff
@@ -4,7 +4,8 @@
   "description": "web scrappers goal",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon -e js server.js"
```

Comment:
Nodemon is great! It's best practice to create an additional `start:dev` script, since, if you ever deployed, tools like Heroku depend on using the `start` script.

So: 
```
"scripts": {
  "start": "node -e js server.js"
   "start:dev": "nodemon -e js server.js"
}
```